packages/cli/__tests__/init.test.ts

```diff
@@ -9,6 +12,29 @@ import { commands } from '../src/commands';
 import { setupTests, TestEnvironment, TestFixture } from '../test-utils';
 
 const beforeEachSetup = setupTests();
+const DEFAULT_REPO = 'https://github.com/launchql/pgpm-boilerplates.git';
+const addDefaults = (argv: ParsedArgs): ParsedArgs => {
+  const defaults = {
+    fullName: 'Tester',
+    email: 'tester@example.com',
+    moduleName: argv.workspace ? 'starter-module' : argv.name || argv.MODULENAME || 'module',
```

Comment:
again, same comment as before. THERE IS NO REASON FOR CAPS.

There is no reason to have all the `||`